### PR TITLE
chore: Migrate NFTListingGrid from vanilla-extract to styled-components

### PR DESCRIPTION
**Pull request by cbachmeier**
No functional difference, should only deprecate use of VE in this file and delete unused style.

### CODE REVIEW

File: src/nft/components/profile/list/NFTListingsGrid.tsx [hunk 1]
```diff
@@ -2,16 +2,19 @@ import { Trans } from '@lingui/macro'
...
 import { DropDownOption, ListingMarket } from 'nft/types'
 import { useMemo, useState } from 'react'
-import styled from 'styled-components/macro'
+import styled, { css } from 'styled-components/macro'
+import { BREAKPOINTS } from 'theme'
 
-import * as styles from './ListPage.css'
 import { NFTListRow } from './NFTListRow'
 
+const GridWrapper = styled.div`
```
Comment (JackShort) [inline src/nft/components/profile/list/NFTListingsGrid.tsx:13]: fyi: there is a `Column` component in `components/Column`

File: src/nft/components/profile/list/NFTListingsGrid.tsx [hunk 2]
```diff
@@ -53,59 +118,30 @@ export const NFTListingsGrid = ({ selectedMarkets }: { selectedMarkets: ListingM
...
-
-          <Column
-            className={bodySmall}
-            color="textSecondary"
-            flex="1"
-            display={{ sm: 'none', lg: 'flex' }}
-            textAlign="right"
-          >
+          </DropdownWrapper>
+
```
Comment (JackShort) [inline src/nft/components/profile/list/NFTListingsGrid.tsx:137]: are these whitespaces necessary?

**Review by JackShort — APPROVED** [review]
lgtm

File: src/nft/components/profile/list/NFTListingsGrid.tsx (hunk 2 above)
Comment (cbachmeier) [reply src/nft/components/profile/list/NFTListingsGrid.tsx:132]: I wanted to separate the functional dropdown vs the static text headers to increase readability

**Pull request merged**